addons/website_slides_survey/data/mail_template_data.xml

```diff
@@ -15,7 +15,7 @@
         Don't hesitate to enroll again!
         <div style="margin: 16px 0px 16px 0px;">
             <a t-att-href="(object.slide_partner_id.channel_id.website_url)"
-                style="background-color: #875A7B; padding: 8px 16px 8px 16px; text-decoration: none; color: #fff; border-radius: 5px; font-size:13px;">
+                t-attf-style="background-color: {{object.survey_id.user_id.company_id.email_secondary_color or '#875A7B'}}; padding: 8px 16px 8px 16px; text-decoration: none; color: #fff; border-radius: 5px; font-size:13px;">
```

Comment:
can we use `user.company_id`

Reply:
Helloo @thsh-odoo 
Changes are done :blush: